source/educators/references/course_development/troubleshoot_video.rst

```diff
@@ -19,7 +19,7 @@ When you use video, you might experience one of the following problems.
     video elements`_.
 
   * Verify that file metadata, particularly the MIME type, is correctly set on
-    the host site. For example, when edX offered support for Internet Explorer
+    the host site. For example, when Open edX platform offered support for Internet Explorer
```

Comment:
```suggestion
    the host site. 
```

I would entirely remove this example,  because we no longer support IE 10 and I think it's more confusing than helpful.